scripts/install.sh

```diff
@@ -183,27 +183,38 @@ cargo install cargo-expand
 
 rustup component add rustfmt
 
-echo "📥 Downloading Node.js"
-download_and_extract \
-    "node-v${NODE_VERSION}-${ARCH_SUFFIX_NODE}.tar.gz" \
-    "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-${ARCH_SUFFIX_NODE}.tar.gz" \
-    z \
-    "${PREFIX}" \
-    1
+
+if ! command -v node &>/dev/null
```

Comment:
The reason why I didn't do this when writing this script is to enforce one Node.js versions for everyone. For example, when you use Node.js 20 or 21 (or in general - not LTS), all our tests break.

And in general, the purpose of this script was version enforcment and solving the dependency hell.

Any reason you are doing this? Is downloading Node.js slow somehow?